### DISCUSSION TITLE
swagger

### DISCUSSION THREAD

**Discussion by TaherBarakat** [realworld-apps/realworld#1559] [General]
i cant find the swagger docs anymore, how can i find it ?

**Comment by geromegrignon** [comment]
Hi @TaherBarakat, it's deactivated currently due to some limitations added on API access. it'll be back soon.

**Reply by geromegrignon** [reply]
You can still use the https://github.com/gothinkster/realworld/blob/main/api/openapi.yml file in Swagger UI meanwhile
